.travis.yml

```diff
@@ -22,19 +22,21 @@ sudo: required
 services:
   - docker
 
-language: generic
+matrix:
+  include:
+  - language: generic
+    env:
+      JAVA=8
 
-env:
-  - JAVA=8
-  - JAVA=11
+  - language: generic
+    env:
+      JAVA=11
 
-before_install:
-  - sudo apt-get -q update
-  - sudo apt-get -q install --no-install-recommends -y curl git gnupg-agent locales pinentry-curses pkg-config rsync software-properties-common
-  - sudo apt-get -q clean
-  - sudo rm -rf /var/lib/apt/lists/*
-  - curl -L https://www-us.apache.org/dist/yetus/0.8.0/yetus-0.8.0-bin.tar.gz | tar xvz -C /tmp/
+  - os: windows
+    language: bash
 
-script:
-  - sed -i.bak 's/openjdk:8/openjdk:'${JAVA}'/' share/docker/Dockerfile
-  - /tmp/yetus-0.8.0/bin/test-patch --plugins=buildtest --java-home=/usr/local/openjdk-${JAVA} --user-plugins=share/precommit/ --run-tests --empty-patch --docker --dockerfile=share/docker/Dockerfile --dirty-workspace --verbose=true
+before_install: "if [ -f ./.travis/before_install.sh   ] ; then ./.travis/before_install.sh ; fi"
```

Comment:
Better to test if the file is executable, rather than just that it exists.

```sh
[ -x ./.travis/before_install.sh ] && ./.travis/before_install.sh
```
etc